VitoProgramming/smart-expense
# Feature/receipts/crud

# Description
1. Complete the initial layout of receipt page
2. The buttons of income and expense can be directed to the relevant pages (URL will be updated as well)
3. The tabs of category and details can be directed to the relevant pages (URL will be updated as well)
4. Add icon can be directed to the receipt-adding page

# Test Plan
1. Access URL '/dashboard/receipts-income-category'
2. Click the buttons of income and expense to see the relevant contents
3. Click the add icon to the receipt-adding page
4. Click '完成' to the receipt to firestore
